src/extension/background.js

```diff
@@ -62,7 +62,7 @@ const storeTheme = ({ theme }) => browser.storage.local.set({ theme });
 const applyTheme = ({ theme }) => {
   log("applyTheme", theme);
   if (!theme) {
-    return;
```

Comment:
So if there's no theme when applyTheme is called we just open a Color Tab ... this should handle the case where the user installs from Test Pilot